src/core/Akka.Cluster/ClusterDaemon.cs

```diff
@@ -1015,7 +1015,7 @@ public ClusterCoreDaemon(IActorRef publisher)
             _cluster = Cluster.Get(Context.System);
             _publisher = publisher;
             SelfUniqueAddress = _cluster.SelfUniqueAddress;
-            _vclockNode = new VectorClock.Node(VclockName(SelfUniqueAddress));
+            _vclockNode = VectorClock.Node.Create(VclockName(SelfUniqueAddress));
```

Comment:
@nvivo yeah, all of the above. If you had smooth sailing in the cluster and there were never any issues with nodes going unreachable this wouldn't be a big problem as the leader's vectorclock was the only one that mattered. 

But as soon as unreachabile events took place then every other individual member node would stamp their information on the `VectorClock` , that would cause the gossips to get corrupted if any of those nodes left the cluster, the end result being that in some cases older gossip appeared to be newer than the most recent gossip produced by the leader. So this affected everything that required cluster convergence. Nasty bug.